Fix signaling-sever package dev build

## Description

Fix issue when you call `yarn dev`. `Signaling-server` package is not beeing builded and command fails.

```shell
web-example:dev: info  - SWC minify release candidate enabled. https://nextjs.link/swcmin
web-example:dev: info  - Disabled SWC as replacement for Babel because of custom Babel configuration ".babelrc" https://nextjs.org/docs/messages/swc-disabled
web-example:dev: info  - Using external babel configuration from /Users/petr.chalupa/Code/react-rtc/apps/web-example/.babelrc
signaling:dev: /Users/petr.chalupa/Code/react-rtc/node_modules/ts-node/src/index.ts:859
signaling:dev:     return new TSError(diagnosticText, diagnosticCodes, diagnostics);
signaling:dev:            ^
signaling:dev: TSError: ⨯ Unable to compile TypeScript:
signaling:dev: src/index.ts(2,39): error TS2307: Cannot find module '@torolocos/signaling-server' or its corresponding type declarations.
signaling:dev: 
signaling:dev:     at createTSError (/Users/petr.chalupa/Code/react-rtc/node_modules/ts-node/src/index.ts:859:12)
signaling:dev:     at reportTSError (/Users/petr.chalupa/Code/react-rtc/node_modules/ts-node/src/index.ts:863:19)
signaling:dev:     at getOutput (/Users/petr.chalupa/Code/react-rtc/node_modules/ts-node/src/index.ts:1077:36)
signaling:dev:     at Object.compile (/Users/petr.chalupa/Code/react-rtc/node_modules/ts-node/src/index.ts:1433:41)
signaling:dev:     at Module.m._compile (/Users/petr.chalupa/Code/react-rtc/node_modules/ts-node/src/index.ts:1617:30)
signaling:dev:     at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
signaling:dev:     at Object.require.extensions.<computed> [as .ts] (/Users/petr.chalupa/Code/react-rtc/node_modules/ts-node/src/index.ts:1621:12)
signaling:dev:     at Module.load (internal/modules/cjs/loader.js:950:32)
signaling:dev:     at Function.Module._load (internal/modules/cjs/loader.js:790:12)
signaling:dev:     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12) {
signaling:dev:   diagnosticCodes: [ 2307 ]
signaling:dev: }
signaling:dev: error Command failed with exit code 1.
signaling:dev: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
signaling:dev: ERROR: command finished with error: command (apps/signaling) yarn run dev exited (1)
command (apps/signaling) yarn run dev exited (1)
```

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   ],
   "scripts": {
     "build": "turbo run build",
-    "dev": "turbo run dev --parallel",
+    "dev": "turbo run dev",
     "lint": "turbo run lint --parallel",
     "tsc:check": "turbo run lint --parallel",
     "test": "turbo run test --parallel",

--- a/turbo.json
+++ b/turbo.json
@@ -5,6 +5,7 @@
       "dependsOn": ["lint"]
     },
     "dev": {
+      "dependsOn": ["^build"],
       "cache": false
     },
     "lint": {},

